instatag.install

```diff
@@ -25,8 +25,9 @@ function instatag_install() {
 }
 
 /**
- * Set instatag snippet url to https://script.bi-instatag.com.
+ * Set instatag snippet url to default content.
  */
 function instatag_update_7001() {
-  variable_set('instatag_snippet_url', 'https://script.bi-instatag.com');
+  $content = '(function(){var script=document.createElement("script");script.src=`https://script.bi-instatag.com?ref=${encodeURIComponent(window.location.href)}`;document.head.appendChild(script);})();';
+  variable_set('instatag_snippet_url', $content);
```

Comment:
@abopche 
the variable name instatag_snippet_url can't be used, as the content it holds now, is not a url anymore.
suggestion,
- do variable_del('instatag_snippet_url')
- use a new variable called ('instatag_snippet')
- We have to set the variable not only in instatag_update_7001, but also in hook_install. This is because, hook_update_7001 won't be called for new installations.